sources/packages/forms/src/form-definitions/reportscholasticstandingchange.json

```diff
@@ -60,56 +82,65 @@
         "multiple": false,
         "unique": false
       },
-      "conditional": {
-        "show": null,
-        "when": null,
-        "eq": ""
-      },
-      "overlay": {
-        "style": "",
-        "left": "",
-        "top": "",
-        "width": "",
-        "height": ""
-      },
       "allowCalculateOverride": false,
       "encrypted": false,
       "showCharCount": false,
       "showWordCount": false,
-      "properties": {},
       "allowMultipleMasks": false,
       "addons": [],
-      "tag": "p",
-      "id": "ey28c8s"
+      "id": "esqfkxi"
     },
     {
       "label": "Application Number | Location Name | Study Dates",
+      "labelWidth": "",
+      "labelMargin": "",
+      "tag": "p",
+      "className": "",
       "attrs": [
         {
           "attr": "",
           "value": ""
         }
       ],
-      "content": "<span class=\"label-bold\">Application #: </span><span class=\"label-value\">{{ data.applicationNumber}}</span><span class=\"muted-content label-value\"> | </span><span class=\"label-bold\">Location: </span><span class=\"label-value\">{{ data.applicationLocationName}}</span>\n",
+      "content": "  <span class=\"label-bold\">Application #: </span\r\n  ><span class=\"label-value\">{{ data.applicationNumber }}</span\r\n  ><span class=\"muted-content label-value\"> | </span\r\n  ><span class=\"label-bold\">Location: </span\r\n  ><span class=\"label-value\">{{ data.applicationLocationName }}</span>",
```

Comment:
extra space just before `span`

Reply:
Not sure how did that space came. But space removed.